Backend/FileReader.cs

```diff
@@ -79,64 +79,126 @@ internal User GetUserData(string username)
             return null;
         }
 
-        internal Task<List<Car>> GetAllCarData()
+        //internal Task<List<Car>> GetAllCarData()
```

Comment:
Maybe these lines should be deleted as commented code can be reached through version control :)